flight/targets/aq32/board-info/pios_board.h

```diff
@@ -133,6 +133,7 @@ extern uintptr_t pios_com_storm32bgc_id;
 #define PIOS_COM_FRSKY_SPORT            (pios_com_frsky_sport_id)
 #define PIOS_COM_OPENLOG                (pios_com_openlog_logging_id)
 #define PIOS_COM_STORM32BGC             (pios_com_storm32bgc_id)
+#define PIOS_COM_OPENLOG                (pios_com_openlog_logging_id)
 
```

Comment:
sorry, i hacked this together and I'm apparently blind.

sx-70 was saying it doesn't work on IRC

Have you seen openlog work on aq32?